spec/requests/resources_spec.rb

```diff
@@ -31,6 +31,23 @@
         expect(returned_ids).to match_array(resources_a.map(&:id))
       end
     end
+
+    context 'with a category_id and latitude/longitude' do
+      close = 10, far = 100, further = 200
+      let!(:category) { create :category, name: 'a' }
+      let!(:resources) do
+        create_list :resource, 3, categories: [category]
+      end
+      let!(:address_further) { create :address, latitude: further, longitude: further, resource_id: resources[0].id }
```

Comment:
I _think_ you should be able to refer to the resource directly and `FactoryGirl` will figure it out. Like `create :address, latitude: further, longitude: further, resource: resources[0]`.